src/screens/App.tsx

```diff
@@ -1,23 +1,95 @@
-import { useState } from "react";
-
+import {
+  ButtonLogin,
+  ButtonCreate,
+  ButtonRegister
+} from "../components/Button/Button";
+import { FeatureTitle, FeatureDescription } from "../components/Text/Text";
 export function App() {
-  const [count, setCount] = useState(0);
-
   return (
-    <main className="flex h-screen w-full flex-col items-center justify-center gap-2 bg-black">
-      <h1 className="text-xl text-white">
-        Vite + React + Typescript + TailwindCSS | Hi
-      </h1>
-      <button
-        type="button"
-        className="bg-white p-2 text-black"
-        onClick={() => {
-          setCount((count) => count + 1);
-        }}
-      >
-        Increase
-      </button>
-      <p className="text-white">Count: {count}</p>
+    <main className="h-screen w-full">
+      <div className="min-h-1/2 bg-blue-600 pb-16">
+        <div className=" mx-auto max-w-screen-2xl">
+          <div className="left-4 top-4 flex items-center font-bold text-white sm:left-8 md:left-12 lg:left-16 xl:left-20">
+            <img
+              src="/Logos/logo.png"
+              alt=""
+              className="mr-2 h-16 w-16 rounded-md border border-gray-500"
+            />
+            <p className="ml-2 text-2xl">CapyFile</p>
+            <div className="flex flex-grow justify-end gap-4">
+              <ButtonLogin text="Login" type="anchor" />
+              <ButtonRegister text="Register" type="anchor" />
+            </div>
+          </div>
+
+          <div className=" top-38 left-4 m-4 mt-16 font-bold text-white sm:left-8 md:left-12 lg:left-16 xl:left-20">
+            <h1 className="text-5xl">
+              Access your files
+              <br />
+              everywhere
+            </h1>
+            <div className="mt-14">
+              <h1 className="text-2xl">
+                Effortlessly store, organize and access your
+                <br /> files with built-in redundancy.
+              </h1>
+            </div>
+            <div className="mt-16">
+              <ButtonCreate
+                text="Create an Account"
+                type="anchor"
+                className="mt-2 "
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="min-h-1/2 relative bg-white">
+        <div className="mx-auto max-w-screen-2xl">
+          <div className=" top-1/5 left-0 mt-12 w-full text-center">
+            <h2 className="text-2xl font-bold text-blue-600">MAIN FEATURES</h2>
+          </div>
+
+          <div className=" left-0 top-1/3 mt-14 flex w-full flex-col items-center  justify-center sm:flex-row">
```

Comment:
This `div` can ve a component (Receive the name of the icon, title and description to show as props). Also, I think it is not worth for the `FeaatureTitle` and `FeatureDescription` to be a component.